core/trino-spi/pom.xml

```diff
@@ -176,6 +176,7 @@
                     </execution>
                 </executions>
                 <configuration>
+                    <versionFormat>[0-9]+</versionFormat>
```

Comment:
That’s too restrictive. What’s the purpose? It will cause problems for anyone doing their own internal releases and tagging them with their own custom version scheme.

Reply:
to me, "doing their own internal releases" implies maintaining a fork, presumably a running fork
in such case, it's trivial to adjust this line appropriately, right?

Reply:
I will close it. It is not super useful in typical case when we do not expect local maven repo to be polluted with some Trino releases which are versione differently to what trinodb/trino does.
Yet I would not say that breaking "anyone doing their own internal releases" is a sound argument. If we really cared about that it would mean we shoud not change **anything** in the codebase, because someone may assume that would not change and their private fork would break due to the change.